packages/nuxt/test/treeshake-client.test.ts

```
@@ -81,16 +81,7 @@ async function SFCCompile (name: string, source: string, options: Options, ssr =
   // eslint-disable-next-line @typescript-eslint/no-unsafe-function-type
-  const result = await (plugin.transform! as Function).call({
-    parse: (code: string, opts: any = {}) => Parser.parse(code, {
-      sourceType: 'module',
-      ecmaVersion: 'latest',
-      locations: true,
-      ...opts,
-    }),
-  }, source, name, {
-    ssr,
-  })
+  const result = await (plugin.transform! as Function)(source, name, { ssr })
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)